packages/ui/constants/index.ts

```diff
@@ -58,3 +58,468 @@ export const MINUTES_PER_YEAR = 24 * 365 * 60;
 
 export const UP_LIMIT = 0.005;
 export const DOWN_LIMIT = 0;
+
+// for additional APR for aBNBC in Ankr
+export const aBNBcContractAddress = '0xBb1Aa6e59E5163D8722a122cd66EBA614b59df0d';
+export const aprDays = 7;
+export const aBNBcContractABI = [
```

Comment:
I think we can keep in the ABI here just the method we need, instead of having the whole thing

Reply:
Done